lib/shared/permissions.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 resin.io
```

Comment:
Or you could just edit this file here, so that it doesn't need to be changed in future? ;)